specs/system_config.md

```diff
@@ -89,6 +103,10 @@ The contained log events are filtered and processed as follows:
   - type `1`: `overhead` and `scalar` overwrite, as two packed `uint256` entries.
   - type `2`: `gasLimit` overwrite, as `uint64` payload.
   - type `3`: `unsafeBlockSigner` overwrite, as `address` payload.
+  - type `4`: `l1BasefeeScalar` and `l1BlobBasefeeScalar` overwrite, as two packed `uint32`
```

Comment:
Perhaps it would be good to clarify that the evm will still pad to 32 bytes so the values will be in the least significant bytes tightly packed together in 8 bytes. It could be good to clarify the byte offsets of the values and their endianness

Reply:
What is the rationale for using `encodePacked` when it still pads to 32 bytes? It makes the parsing logic a bit different for this handler compared to the others and future versions of `solc` want to remove `encodePacked`. It would still be possible to tightly pack them without that helper function if we did want to upgrade but just wondering since it doesn't save any data size

Reply:
my understanding was if we used regular abi.Encode it would be 32 bytes per uint for 64 bytes total.

Reply:
ah you are correct